tests/cli/conftest.py

```diff
@@ -55,3 +71,34 @@ def do_run(*args):
         return testdir.run(*args)
 
     return do_run
+
+
+def _set_up_initial_project(testdir: Testdir):
+    # copy initial project files
+    testdir.copy_example("rasa/cli/initial_project/actions.py")
+    testdir.copy_example("rasa/cli/initial_project/credentials.yml")
+    testdir.copy_example("rasa/cli/initial_project/domain.yml")
+    testdir.copy_example("rasa/cli/initial_project/endpoints.yml")
+    testdir.mkdir("data")
+    testdir.copy_example("rasa/cli/initial_project/data")
+    testdir.run("mv", "nlu.md", "data/nlu.md")
+    testdir.run("mv", "stories.md", "data/stories.md")
+
+    # create a config file
+    # for the cli test the resulting model is not important, use components that are
+    # fast to train
+    write_yaml_file(
```

Comment:
why do we not use the default one and reuse the model we already have trained for this?

Reply:
This is used for cli tests that do not need a model on start. They would train a model inside the test. We cannot reuse the model in those cases. We update the config file here to reduce the training time of those tests.